Extending/Backoffice-Tours/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 versionFrom: 9.0.0
+versionTo: 10.0.0
```

Comment:
I guess this needs to be removed again if we update this article to make it work for 10.
As I guess the backoffice tours are not meant to leave?

Reply:
I will merge this PR in now as is.

I have a task on my board, on running some more tests with these tours - in relation to the tests we ran at our previous meeting.